vizro-core/pyproject.toml

```diff
@@ -35,8 +35,7 @@ requires-python = ">=3.8"
 
 [project.optional-dependencies]
 kedro = [
-  "kedro>=0.17.3",
-  "kedro-datasets",  # no longer a dependency of kedro for kedro>=0.19.2
+  "kedro>=0.19.3",  # pinned by Snyk to avoid a vulnerability: https://security.snyk.io/vuln/SNYK-PYTHON-KEDRO-6370687
```

Comment:
Took out `kedro-datasets` given that we need to upgrade to `kedro>=0.19.2`. Not sure whether we actually want to do this? Where do we actually stand on the whole snyk discussion @antonymilne ? 🤔  Do we continue merging these proposals by snyk or do we selectively choose which ones we include?

Reply:
In general I'm working out a scheme for handling these better, which will basically mean ignoring most, if not all, of snyk's suggested bumps or turning off the feature altogether.

Before that's ready, let's continue merging them where it's easy and looks sensible to do so and ignore them where it doesn't. This is definitely a case where it doesn't. Let's just ignore the PR for now and I'll handle it in due course...